presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -10,50 +10,67 @@ Additional types can be provided by plugins.
     Connectors are not required to support all types.
     See connector documentation for details on supported types.
 
-BOOLEAN
--------
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Boolean Type
```

Comment:
I think you still might want to call this 'boolean types' - weird to have a heading that's singular..